### ISSUE TITLE
x-only BIP-Schnorr?

### ISSUE THREAD

**Issue by ChristopherA** [apoelstra/sighacker#3]
@apoelstra — any thoughts on updating this tool to use @ncklr's new x-only BIP-Schorr in secp256k1 library's PR https://github.com/bitcoin-core/secp256k1/pull/558?

What do you suggest the best way to do this as I don't think you can git submodule a PR.

I know that I'd like to see the equivalent of minisign (file signing) and minilock (file encryption to groups) for x-only SecP Schnorr.

-- Christopher Allen

**Comment by apoelstra** [comment]
Yeah, I'd really like to see that. Unlikely I'll have time before that PR is merged though.

**Comment by ChristopherA** [comment]
Is it expected that it will be merged soon? I thought it wouldn't happen until at least BIP-Schnorr had a real BIP number, and then likely one or more Bitcoin-Core release cycles.

-- Christopher Allen

**Comment by apoelstra** [comment]
Yeah, it won't be merged until it has a clear usecase in Core. So fairly late stages of Taproot integration.
